offload-build.in

```diff
@@ -114,14 +115,17 @@ mapfile -t files < <(
             if [[ -f /usr/share/devtools/makepkg-'"${repo}"'-'"${arch}"'.conf ]]; then
                 makepkg_config="/usr/share/devtools/makepkg-'"${repo}"'-'"${arch}"'.conf"
             fi &&
-            makepkg --config <(cat "${makepkg_user_config}" "${makepkg_config}" 2>/dev/null) --packagelist
+            makepkg --config <(cat "${makepkg_user_config}" "${makepkg_config}" 2>/dev/null) --packagelist &&
+            printf "%s/%s\n" "${temp}" "PKGBUILD"
```

Comment:
```suggestion
            printf "%s\n" "${temp}/PKGBUILD"
```

This is more readable because you don't need to figure out how printf is joining the multiple arguments. The path separator isn't variable since this isn't a multi-platform Windows-supporting API like python `os.path.join()`, so let's make printf do one thing (emit stuff to stdout) and provide the entire string as one already-formatted argument.

Reply:
sounds fine as well